pymeasure/adapters/prologix.py

```diff
@@ -66,10 +67,14 @@ class PrologixAdapter(SerialAdapter):
 
     def __init__(self, port, address=None, rw_delay=None, serial_timeout=0.5,
                  preprocess_reply=None, **kwargs):
+        kwargs.setdefault('write_termination', "\n")
```

Comment:
Is that the default, i.e. not a behaviour change?

Reply:
Before, '\n' was hard coded as term char into the read and write. Therefore this default maintains the same functionality, but allows also to change it, if desired.

I made sure, to keep the behavior the same.